packages/grid/x-grid-modules/src/components/styled-wrappers/GridRootStyles.ts

```diff
@@ -32,10 +34,14 @@ export const useStyles = makeStyles(
           top: 0,
           left: 0,
           right: 0,
-          bottom: 15,
+          bottom: 0,
           alignSelf: 'center',
           alignItems: 'center',
           zIndex: 10,
+          backgroundColor: fade(
+            theme.palette.background.default,
+            theme.palette.action.disabledOpacity,
+          ),
```

Comment:
Add missing color indicator that there is an overlay blocking the interaction on the cells.